src/back-end/lib/permissions.ts

```diff
@@ -165,6 +165,10 @@ export async function readOneCWUProposal(connection: Connection, session: Sessio
         (session.user && await isCWUProposalAuthor(connection, session.user, proposalId)) || false;
 }
 
+export async function canSeeCWUProposalScore(connection: Connection, session: Session, opportunityId: string): Promise<boolean> {
+  return isAdmin(session) || (session.user && await isCWUOpportunityAuthor(connection, session.user, opportunityId) || false);
+}
+
```

Comment:
This function's name and implementation are misleading, as a vendor can see their own proposals' scores and ranks once they have been awarded/notAwarded. Can you please update accordingly?

Reply:
I wasn't aware vendors would be able to see their scores at any point (this is different than the original devex).  I will update the function.

Reply:
Okay I've updated the function: `readCWUProposalScore` and it allows vendors to view their score and rank if the proposal status is in Awarded/NotAwarded (or if the user is Admin/Opp Author).

I also realized we weren't restricting score based on user type/proposal status, so I've added a check there as well (I think you added another ticket for this).